### PR TITLE
Fix bucket metrics api path in docs

### PR DESCRIPTION
**Pull request by vanugrah**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
Corrects a small error in the docs.

## Motivation and Context
It's nice to have bucket metrics :) 

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

File: docs/metrics/prometheus/list.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ Metrics may include one or more additional labels, such as the server that calcu
 These metrics can be obtained from any MinIO server once per collection by using the following URL:
 
 ```shell
-https://HOSTNAME:PORT/minio/metrics/v2/bucket
+https://HOSTNAME:PORT/minio/v2/metric/bucket
```
Comment (vadmeste) [inline docs/metrics/prometheus/list.md]: Shouldn't it be 'metrics' instead of 'metric' ?

**Review by vadmeste — CHANGES_REQUESTED** [review]

File: docs/metrics/prometheus/list.md (hunk 1 above)
Comment (vanugrah) [reply docs/metrics/prometheus/list.md]: Fixed

**Review by vadmeste — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by vanugrah** [comment]
For some reason the build on windows is failing. Likely unrelated to this change. Let me know if anything else is required from me.

**Pull request merged**